server/src/termdb.barchart.js

```diff
@@ -634,9 +634,20 @@ Output: The function has no return but appends the statistical results to the pr
 	]
 }
 */
-async function computePvalues(data, ds) {
+async function computePvalues(data, ds, hiddenValues) {
+	// remove hidden values from deep copy of data
+	const charts = JSON.parse(JSON.stringify(data.charts))
+	for (const chart of charts) {
+		chart.serieses = chart.serieses.filter(s => !hiddenValues.term1.includes(s.seriesId))
```

Comment:
I don't think you should replace the same array with filtered array. Would be better to create new array.